Parse/Internal/File/NikitaInputStream/NikitaInputStream.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*!
+ @abstract Why? Because Nikita.
+ @discussion Actually, this does have some practical use. When downloading a file stream via 
```

Comment:
I don't remember precisely, but I think these annotations are simply ignored for class documentation.